.github/workflows/ci.yaml

```diff
@@ -66,61 +59,65 @@ jobs:
     steps:
       - name: Checkout code from the repository
         uses: actions/checkout@v4
-
       - name: Set up node.js
         uses: actions/setup-node@v4
         with:
           node-version: ${{ inputs.node_version }}
-
       - name: Install clean dependencies
         run: npm ci
-
       - name: Run headless tests
         id: run-tests
-        run: npm run test -- --watch=false --browsers=ChromeHeadless 
-
+        run: npm run test -- --watch=false --browsers=ChromeHeadless
       - name: Build angular project
         id: build-project
         run: npm run build
-
-      - name: Log in to github container registry  
+      - name: Log in to github container registry
         id: login-registry
-        run: | 
+        run: |
           echo "${{ secrets.github_token }}" \
           | docker login ${{ inputs.docker_registry }} -u ${{ github.actor }} --password-stdin
-
       - name: Build docker image
         id: build-docker
         run: docker build -t $IMAGE_NAME .
         working-directory: .
-
       - name: Push docker image
         id: push-docker
         run: docker push $IMAGE_NAME
-
       - name: Get pipeline execution time
         id: get-time
         if: always()
         env:
           GH_TOKEN: ${{ github.token }}
-        run: echo "execution_time=$(gh run list --json createdAt,updatedAt -w ci.yaml -L 1 | jq -r '.[0] | (( (strtotime(.updatedAt) - strtotime(.createdAt)) ))')s" >> $GITHUB_ENV
-
+        run: echo "execution_time=$(gh run list --json createdAt,updatedAt -w ci.yaml
+          -L 1 | jq -r '.[0] | (( (strtotime(.updatedAt) - strtotime(.createdAt))
+          ))')s" >> $GITHUB_ENV
+      - name: Unlock secrets with git-crypt
+        if: always()
+        uses: sliteteam/github-action-git-crypt-unlock@1.2.0
+        env:
+          GIT_CRYPT_KEY: ${{ secrets.GIT_CRYPT_KEY }}
+      - name: Read secrets from secrets.yaml
+        id: read-secrets
+        if: always()
+        run: |
+          cat secrets.txt >> "$GITHUB_OUTPUT"
```

Comment:
I don't see that you use anywhere vars from your secrets in this CI/CD

Reply:
Fixed